README.md

```diff
@@ -1,71 +1,120 @@
 
+Now your tests can work without any internet connection\!
 
-Record HTTP requests and responses on disk and replay them for the unit tests of your R package to make them independent from any connection, faster, and more complete. An R port of the Ruby gem [vcr](https://github.com/vcr/vcr)
+[Demo of adding vcr testing to an R
+package](https://github.com/maelle/exemplighratia/pull/2/files)
 
 ## Docs
 
-Check out the [HTTP testing book](https://books.ropensci.org/http-testing) and the [vcr vignettes](https://docs.ropensci.org/vcr/articles/).
+Check out the [HTTP testing
+book](https://books.ropensci.org/http-testing) and the [vcr
+vignettes](https://docs.ropensci.org/vcr/articles/).
 
 ## Supported HTTP libraries
 
-* [crul](https://github.com/ropensci/crul)
-* [httr](https://github.com/r-lib/httr)
+  - [crul](https://docs.ropensci.org/crul)
+  - [httr](https://httr.r-lib.org/)
 
 ## Getting Started
 
+The docs assume you are using testthat for your unit tests.
+
 To install from CRAN
 
-```r
+``` r
 install.packages("vcr")
 ```
 
+### `use_vcr`
+
 You can then set up your package to use `vcr` with:
 
-```r
+``` r
 vcr::use_vcr()
 ```
 
 This will:
 
-```
-◉ Using package: vcr.example  
-◉ assuming fixtures at: tests/fixtures  
-✓ Adding vcr to Suggests field in DESCRIPTION  
-✓ Creating directory: ./tests/testthat  
-◉ Looking for testthat.R file or similar  
-✓ tests/testthat.R: added  
-✓ Adding vcr config to tests/testthat/setup-vcr.example.R  
-✓ Adding example test file tests/testthat/test-vcr_example.R  
-✓ .gitattributes: added  
-◉ Learn more about `vcr`: https://books.ropensci.org/http-testing
-```
-
-If you need to use secrets (for example, api keys for authentication), it's important to protect them. `vcr` saves responses from API's as YAML files, and this will include your secrets unless you indicate to `vcr` what they are and how to protect them. This is one of the uses  of the `setup-vcr` file that got made for us just now. By default it will look like this:
-
-```r
+  - put `vcr` into the `DESCRIPTION`
+  - check that `testthat` is setup
+  - setup `testthat` if not
+  - set the recorded cassettes to be saved in and sourced from
+    `tests/fixtures`
+  - setup a config file for `vcr`
+  - add an example test file for `vcr`
+  - make a `.gitattributes` file with settings for `vcr`
+  - make a `./tests/testthat/setup-vcr.R` file
+
+What you will see in the R console:
+
+    ◉ Using package: vcr.example  
+    ◉ assuming fixtures at: tests/fixtures  
+    ✓ Adding vcr to Suggests field in DESCRIPTION  
+    ✓ Creating directory: ./tests/testthat  
+    ◉ Looking for testthat.R file or similar  
+    ✓ tests/testthat.R: added  
+    ✓ Adding vcr config to tests/testthat/setup-vcr.example.R  
+    ✓ Adding example test file tests/testthat/test-vcr_example.R  
+    ✓ .gitattributes: added  
+    ◉ Learn more about `vcr`: https://books.ropensci.org/http-testing
+
+### Protecting secrets
```

Comment:
the docs about secrets were slightly different in the README and vignette, I tried merging them.